runtime_config_linux.go

```diff
@@ -216,6 +222,8 @@ type Resources struct {
 	HugepageLimits []HugepageLimit `json:"hugepageLimits,omitempty"`
 	// Network restriction configuration
 	Network *Network `json:"network,omitempty"`
+	// IntelRdt restriction configuration
+	IntelRdt IntelRdt `json:"intelRdt,omitempty"`
```

Comment:
I think it's better to use pointer to be consistent with other fields, you can see from https://github.com/opencontainers/specs/pull/233 why we are doing this.